src/components/Table/GridTable.tsx

```diff
@@ -1108,129 +1126,146 @@ function GridRow<R extends Kinded, S>(props: GridRowProps<R, S>): ReactElement {
 
   let firstContentColumnStylesApplied = false;
 
-  const rowNode = (
-    <Row css={rowCss} {...others} data-gridrow {...getCount(row.id)}>
-      {columns.map((column, columnIndex) => {
-        const { wrapAction = true, isAction = false } = column;
+  const rowNode = useComputed(
```

Comment:
Why the `useComputed`? I suppose it's b/c we're accessing `rowState.activeCellId`...

In theory `GridTable` will also do this if `GridTable.observeRows` is set (i.e. wrap `GridRow` in an `ObservableGridRow`)...

Maybe we should change:

```
      const RowComponent = observeRows ? ObservedGridRow : MemoizedGridRow;
```

To be `observeRows || cellHighlight ? ... :`

Or we could just _always_ use `ObservedGridRow`, it probably wouldn't hurt anything, and we could remove the `observeRows` prop all together (b/c it's always set true now); maybe just do that?

Reply:
by doing this, removing the observeRows prop, are we introducing a breaking change on internal-frontend?

Reply:
@bdow what do you think?

Reply:
I kind of like the idea of just _always_ doing `observeRows`. Maybe for now keep the prop, but default it to be `true`. This should not be a breaking change on BP.

Reply:
I'm also good with removing the prop too... Would just mean we need to clean up places that set it, which would be an easy fix.